fdbserver/ClusterController.actor.cpp

```diff
@@ -3591,6 +3591,7 @@ ACTOR Future<Void> clusterRecruitFromConfiguration(ClusterControllerData* self,
 				return Void();
 			} else if (e.code() == error_code_operation_failed || e.code() == error_code_no_more_servers) {
 				// recruitment not good enough, try again
+				TraceEvent("RecruitFromConfigurationRetry", self->id).error(e);
```

Comment:
Let's also add whether `goodRecruitmentTime` is ready or not in the traceevent

Reply:
And also how long until it is ready.

Reply:
What I meant was to add 

.detail("GoodRecruitmentTimeReady", goodRecruitmentTime.isReady())


Regarding `And also how long until it is ready.`, I was thinking whether we can print something that how long until the goodRecruitmentTime is ready (the amount of time left in the delay), but looking at Future class, I don't see a way to actually get this info, so you can ignore it for now.